tests/SConscript

```diff
@@ -5,7 +5,7 @@ import subprocess
 from lsst.sconsUtils import env, scripts, targets
 import SCons.Script
 
-scripts.BasicSConscript.tests()
+scripts.BasicSConscript.tests(pyList=[])
```

Comment:
That docstring you commented on above explains this. If you say `pyList=[]` that enables automatic test discovery. `pyList=None` invokes old style "find all *.py files". Using `pyList` like this was the simplest way to retain backwards compatibility and let you enable auto discovery. An entirely new argument to enable auto test discovery and disable `pyList` seemed overkill (I did try it that way first).